js/utils/currency.js

```diff
@@ -115,3 +128,105 @@ export function formatCurrency(amount, currency, locale = app.settings.get('lang
 
   return formattedCurrency;
 }
+
+let exchangeRates = {};
+
+/**
+ * Will fetch exchange rate data from the server. This is already called
+ * on an interval via exchangeRateSyncer.js, so it's unlikely you would
+ * need to call this method. Instead access cached values via getExchangeRate()
+ * or more commonly convertCurrency().
+ */
+export function fetchExchangeRates(options = {}) {
+  const xhr = $.get(app.getServerUrl('ob/exchangerates/'), options)
+    .done((data) => (exchangeRates = data));
+
+  events.trigger('fetching-exchange-rates', { xhr });
+
+  return xhr;
+}
+
+export function getExchangeRate(currency) {
+  if (!currency) {
+    throw new Error('Please provide a currency.');
+  }
+
+  return exchangeRates[currency];
+}
+
+export function NoExchangeRateDataError(message) {
+  this.message = message || 'Missing exchange rate data';
+  this.name = 'NoExchangeRateDataError';
+  this.stack = (new Error()).stack;
+}
+
+NoExchangeRateDataError.prototype = Object.create(Error.prototype);
+NoExchangeRateDataError.prototype.constructor = NoExchangeRateDataError;
+
+/**
+ * Converts an amount from one currency to another based on exchange
+ * rate data.
+ */
+export function convertCurrency(amount, fromCur, toCur) {
+  if (typeof amount !== 'number') {
+    throw new Error('Please provide an amount as a number');
+  }
+
+  if (isNaN(amount)) {
+    throw new Error('Please provide an amount that is not NaN');
+  }
+
+  if (typeof fromCur !== 'string') {
+    throw new Error('Please provide a fromCur as a string');
+  }
+
+  if (typeof toCur !== 'string') {
+    throw new Error('Please provide a toCur as a string');
+  }
+
+  if (fromCur === toCur) {
+    return amount;
+  }
+
+  if (!exchangeRates[fromCur]) {
+    throw new NoExchangeRateDataError(`We do not have exchange rate data for ${fromCur}.`);
+  }
+
+  if (!exchangeRates[toCur]) {
+    throw new NoExchangeRateDataError(`We do not have exchange rate data for ${toCur}.`);
+  }
+
+  const fromRate = fromCur === 'BTC' ? 1 : getExchangeRate(fromCur);
+  const toRate = toCur === 'BTC' ? 1 : getExchangeRate(toCur);
+
+  return (amount / fromRate) * toRate;
```

Comment:
When I check the amounts using the app and Google's currency conversion calculator, things check out.

Anyhow, right now our api is returning the following rates:
"ISK": 83179.69,
"USD": 739.07,

So, using the formula in the code, $5 USD would result in 562.73 ISK, which I believe is right.